feat: Add docs about what `captureMessage` does (#11258)
getsentry/sentry-docs: docs/platforms/android/usage/index.mdx

Another common operation is to capture a bare message. A message is textual information that should be sent to Sentry. Typically, our SDKs don't automatically capture messages, but you can capture them manually.

> Messages also show up in your issue stream.

Reviewer: I think this could be clearer. Where in the issue stream does it show up? Is it accurate to say something like:

 ```suggestion
When captured, messages are used as the issue name in the issue stream. Messages are also displayed under "Message" in the Issue Details page. To search and filter issues by messages, use the [`message`](/concepts/search/searchable-properties/issues/#message) property. 
```

Author: Hmm, I don't know, this feels like more details than we have for the capture exception case above? 🤔 

IMHO this:

>  Messages are also displayed under "Message" in the Issue Details page.

sounds a bit redundant to me (we also don't explain that exceptions show up under exception etc, do we?) 🤔 

What about something like this instead, somewhere in the middle:

```suggestion
When captured, messages show up in your issue stream and use the message as the issue name.
```

Reviewer: What about making it even simpler and just saying this:
```suggestion
When captured, messages show up as the issue name.
```

Author: I think this misses the key point I wanted to add here (based on the linked issue) that is, what happens with messages at all? 🤔 

What about:

```suggestion
Messages show up as issues on your issue stream, with their message as the issue name.
```
